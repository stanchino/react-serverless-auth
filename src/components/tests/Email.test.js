import { Email } from "../..";

import { matchFormSnapshot } from "./shared-examples";

describe("Email", () => it("matches the snapshot", () => matchFormSnapshot(Email)));