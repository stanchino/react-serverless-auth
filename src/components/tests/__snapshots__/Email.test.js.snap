// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Email matches the snapshot 1`] = `
<form
  className={undefined}
  onSubmit={[Function]}
>
  <div
    className="messages"
  >
    <div
      className="flash"
    />
  </div>
  <div>
    <label />
    <div>
      <input
        name="email"
        onBlur={[Function]}
        onChange={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        placeholder="Email"
        type="text"
        value=""
      />
    </div>
  </div>
</form>
`;
